### ISSUE TITLE
Bug: infinitely loading progress bar on Copilot action 

### ISSUE THREAD

**Issue by judson-at-pieces** [pieces-app/plugin_sublime#227]
### Bug Description

When selecting any Pieces Copilot action within Pieces for Sublime Text, the newly-added progress bar was stuck in an infinite loop and the copilot action in question (explaining the current code selection) did not complete, though the side-view window opened up.

### Your operating system

MacOS

### What is the command name?

Pieces: Explain

### Your Pieces OS Version `Pieces: About`

10.1.10

### Your Pieces for Sublime plugin version `Pieces: About`

1.4.0

**Comment by bishoy-at-pieces** [comment]
@judson-at-pieces is this happening again?
I think the problem was in PiecesOS since that and the copilot doesn't want to send a request to stop generating

**Comment by nolan-at-pieces** [comment]
@bishoy-at-pieces Yes still happening, was coming on here to make an issue about it now.

**Comment by bishoy-at-pieces** [comment]
Hey @nolan-at-pieces @judson-at-pieces can you tell me what did you exactly do because I can't reproduce that issue
